src/scss/_back-to-top.scss

```diff
@@ -0,0 +1,36 @@
+#return-to-top {
+  position: fixed;
+  bottom: 1.25em; //20px
+  right: 1.25em; //20px
+  background: rgb(0, 0, 0);
```

Comment:
Just curious, is there a reason you are using rgb colors instead of typical hex?